libsolidity/formal/SMTEncoder.h

```diff
@@ -115,6 +115,9 @@ class SMTEncoder: public ASTConstVisitor
 	void compareOperation(BinaryOperation const& _op);
 	void booleanOperation(BinaryOperation const& _op);
 	void bitwiseOperation(BinaryOperation const& _op);
+	void bitwiseNotOperation(UnaryOperation const& _op);
+
+	std::tuple<unsigned, bool> typeBvSizeAndSigndness(TypePointer const& type);
```

Comment:
```suggestion
	std::tuple<unsigned, bool> typeBvSizeAndSignedness(TypePointer const& type);
```

Reply:
Nice! It looks nice now with the helper function. Just one thing, could you move it to `SymbolicTypes.h/cpp`? We have similar helpers there, so I think it would it fit. There the function can be `static`.

Reply:
Ah, any special reason for using `tuple` instead of just `pair`?

Reply:
No special reason at all. Will change to pair.